docs/api.md

```diff
@@ -0,0 +1,233 @@
+
+
+#### OS X/Mac App Store targets only
+
+##### `app-bundle-id`
+
+*String*
+
+The bundle identifier to use in the application's plist.
+
+##### `app-category-type`
+
+*String*
+
+The application category type, as shown in the Finder via *View → Arrange by Application Category* when viewing the Applications directory.
+
+For example, `app-category-type=public.app-category.developer-tools` will set the application category to *Developer Tools*.
+
+Valid values are listed in [Apple's documentation](https://developer.apple.com/library/ios/documentation/General/Reference/InfoPlistKeyReference/Articles/LaunchServicesKeys.html#//apple_ref/doc/uid/TP40009250-SW8).
+
+##### `extend-info`
+
+*String*
+
+Filename of a plist file; the contents are added to the app's plist. Entries in `extend-info` override entries in the base plist file supplied by electron-prebuilt, but are overridden by other explicit arguments such as `app-version` or `app-bundle-id`.
+
+##### `extra-resource`
+
+*String* or *Array*
+
+Filename of a file to be copied directly into the app's `Contents/Resources` directory.
+
+##### `helper-bundle-id`
+
+*String*
+
+The bundle identifier to use in the application helper's plist.
+
+##### `osx-sign`
+
+*Object* or *`true`*
+
+If present, signs OS X target apps when the host platform is OS X and XCode is installed. When the value is `true`, pass default configuration to the signing module. The configuration values listed below can be customized when the value is an `Object`. See [electron-osx-sign](https://www.npmjs.com/package/electron-osx-sign#opts) for more detailed option descriptions and the defaults.
+- `identity` (*String*): The identity used when signing the package via `codesign`.
+- `entitlements` (*String*): The path to the 'parent' entitlements.
+- `entitlements-inherit` (*String*): The path to the 'child' entitlements.
+
+#### Windows targets only
+
+##### `version-string`
+
+*Object*
+
+Object (also known as a "hash") of application metadata to embed into the executable:
+- `CompanyName`
+- `LegalCopyright` (**deprecated** and will be removed in a future major version, please use the top-level `app-copyright` parameter instead)
+- `FileDescription`
+- `OriginalFilename`
+- `FileVersion` (**deprecated** and will be removed in a future major version, please use the top-level `build-version` parameter instead)
+- `ProductVersion` (**deprecated** and will be removed in a future major version, please use the top-level `app-version` parameter instead)
+- `ProductName`
+- `InternalName`
+
+## callback
+
+### `err`
+
+*Error* (or *Array*, in the case of an `ncp` error)
+
+Contains errors, if any.
+
+### `appPath`
+
+*String*
+
+Path to the newly created application.
```

Comment:
Fixed in https://github.com/electron-userland/electron-packager/pull/296/commits/68413f8e3948cef759321510aef5f9eeccad6ea7.